go/ql/lib/semmle/go/dataflow/ExternalFlow.qll

```diff
@@ -96,28 +95,18 @@ private predicate packageLink(string shortpkg, string longpkg) {
   longpkg.prefix(longpkg.indexOf(".")) = shortpkg
 }
 
-/**
- * Holds if `package` has MaD framework coverage and it is not a subpackage of
- * any other package with MaD framework coverage.
- */
 private predicate canonicalPackage(string package) {
   relevantPackage(package) and not packageLink(_, package)
 }
 
-/**
- * Holds if `package` and `subpkg` have MaD framework coverage, `subpkg` is a
- * subpackage of `package` (or they are the same), and `package` is not a
- * subpackage of any other package with MaD framework coverage.
- */
 private predicate canonicalPackageHasASubpackage(string package, string subpkg) {
   canonicalPackage(package) and
   (subpkg = package or packageLink(package, subpkg))
 }
 
 /**
  * Holds if MaD framework coverage of `package` is `n` api endpoints of the
- * kind `(kind, part)`, and `pkgs` is the number of subpackages of `package`
- * which have MaD framework coverage (including `package` itself).
+ * kind `(kind, part)`.
```

Comment:
Why does this stop documenting `pkgs`, which still exists?

Reply:
To match the java and C# versions of this file, so they're easier to keep in sync or make into a shared library. You're right that it's better though. Maybe I'll keep it and make a tiny PR to add it to the others.

Reply:
Done here: https://github.com/github/codeql/pull/16676